addon/package.json

```diff
@@ -63,21 +56,7 @@
       "title": "Allowed origins for addon",
       "description": "Allowed origins for addon",
       "type": "string",
-      "value": "http://ideatown.dev:8000/*,http://localhost:1337/*,http://127.0.0.1:1337/*,https://ideatown.firefox.com/*,https://mozilla.github.io/idea-town-addon/stub-content/,https://people.mozilla.org/~jhirsch/universal-search-addon/*"
-    }
-  ],
-  "UPDATES": [
-    {
-      "id": "universal-search-addon@mozilla.com",
-      "name": "Universal Search",
-      "desc": "Expandable embed search from your address bar",
-      "url": "https://people.mozilla.org/~jhirsch/universal-search-addon/addon.xpi"
-    },
-    {
-      "id": "foxsplitter@piro.sakura.ne.jp",
-      "name": "Fox Splitter",
-      "desc": "Split your browser windows like a tiling wm",
-      "url": "https://addons.mozilla.org/firefox/downloads/latest/324976/addon-324976-latest.xpi?src=search"
+      "value": "http://ideatown.dev:8000/*,https://ideatown.firefox.com/*, https://people.mozilla.org/~jhirsch/universal-search-addon/*"
```

Comment:
Does this need to list a people.mozilla.org URL? My impression was this lists the hosts where the addon will listen to messages from content. Hopefully we don't need to list the URLs of every experiment addon. At launch this should just be ideatown.firefox.com and hopefully a solution for issue #175 will let us easily switch to ideatown.dev